.github/workflows/build.yml

```diff
@@ -171,8 +171,7 @@ jobs:
         run: |
           add-apt-repository ppa:git-core/ppa
           apt-get update
-          apt-get --yes install apt-transport-https ca-certificates git
-          update-ca-certificates
```

Comment:
Was this only needed for tap?

Reply:
Yes - see its addition in #7087. It could technically be useful if we add other submodules in the future, but I'd hope we'd be on a newer Ubuntu version by then, and I'd like to avoid doing unnecessary work in the CI build scripts.